packages/website-eslint/mocks/lru-cache.js

```diff
@@ -0,0 +1,14 @@
+class LruCache {
+  constructor() {
+    this.cache = new Map();
+  }
+  get(name) {
+    return this.cache.get(name);
+  }
+  set(name, value) {
+    this.cache.set(name, value);
+    return value;
+  }
+}
+
+module.exports = LruCache;
```

Comment:
`lru-cache` SADLY dons't workwell in browser, but code can be relatively simplified if we don't care about long persistent runs (all the calls are static)

with that change i was able to stop changing how `semver` behaves and just import it as a whoe library (i noticed that previously not everything was working)